manifests/02_config.crd.yaml-merge-patch

```diff
@@ -0,0 +1,10 @@
+# this file can be removed once we switch to v0.2 of crd generator
+# see: https://github.com/openshift/cluster-kube-scheduler-operator/pull/148
```

Comment:
@damemi stange comment. I don't see the pattern annotation in the api types. Maybe we should add it and get rid of these patches.

Reply:
https://github.com/openshift/api/blob/master/operator/v1/types.go#L27 ?